src/modules/expression-evaluator/src/evaluateExpressionTestData.ts

```diff
@@ -769,94 +744,121 @@ testData.getListOfApplications_withId = {
 testData.simpleGraphQL = {
   operator: 'graphQL',
   children: [
-    {
-      value: `query App($appId:Int!) {
-        application(id: $appId) {
-          name
-        }
-      }`,
-    },
-    { value: ['appId'] },
-    { value: 1000 },
-    { value: 'application.name' },
+    `query App($appId:Int!) {
+      application(id: $appId) {
+        name
+      }
+    }`,
+    'graphQLEndpoint',
+    ['appId'],
+    4000,
+    'application.name',
   ],
 }
 
 testData.GraphQL_listOfApplications = {
   operator: 'graphQL',
   children: [
-    {
-      value: `query Apps {
+    `query Apps {
       applications {
         nodes {
           name
         }
       }
     }`,
-    },
-    { value: [] },
-    { value: 'applications.nodes' },
+    'graphQLEndpoint',
+    [],
+    'applications.nodes',
   ],
 }
 
 testData.GraphQL_listOfApplicationsWithId = {
   operator: 'graphQL',
   children: [
-    {
-      value: `query Apps {
-        applications {
-          nodes {
-            name
-            id
-          }
+    `query Apps {
+      applications {
+        nodes {
+          name
+          id
         }
-      }`,
-    },
-    { value: [] },
-    { value: 'applications.nodes' },
+      }
+    }`,
+    'graphQLEndpoint',
```

Comment:
As we discussed, it'd be good if the empty string "" also defaulted to the internal endpoint, so could you make sure (at least) one of the test cases uses this? (And update the docs)

Reply:
Updated the docs - by committing your suggestion ✅ 
Adding new test to suite 📝

Reply:
Done: 2496d24